### ISSUE TITLE
use monaco set,i18n is not work

### ISSUE THREAD

**Issue by huyanhai** [suren-atoyan/monaco-loader#40]
<img width="1191" alt="image" src="https://github.com/suren-atoyan/monaco-loader/assets/28421572/c96a6773-1b1f-4564-9ea7-15bf74598468">

**Comment by suren-atoyan** [comment]
right, unfortunately, you can't use `vs/nls` in `loader.config` if you use `monaco`
